css/off.canvas.drawer.css

```diff
@@ -0,0 +1,241 @@
+#drawer-button {
+  float:right;
+  color:#000;
+  border:none;
+  background-color:rgba(0, 0, 0, 0);
+  font-family: Arial,Sans-Serif;
+  padding:10px;
+  font-size:14px;
+  line-height: 1.75em!important;
+  outline: 0;
+}
+
+#drawer-button:hover {
+  background-color: #002145!important;
+  color: #fff!important;
+}
+
+.drawer-toggle--hamburger {
+  background-color: #fff;
+  -webkit-box-shadow: 1px 1px 1px #191938;
+  float: right;
+  height: 24px;
+  padding: 11px 0 0 13px;
+  width: 32px;
+}
+
+.drawer-toggle--hamburger,.drawer-toggle--hamburger:hover {
+  box-shadow: 1px 1px 1px #191938;
+  -moz-box-shadow: 1px 1px 1px #191938;
+}
+
+.drawer-toggle--hamburger:hover {
+  -webkit-box-shadow: inset 1px 1px 1px #191938;
```

Comment:
Should this be in the above statement on line 28? It seems like it would only work in webkit on hover.